doc/api/next_api_changes/2017-10-24-DS.rst

```diff
@@ -0,0 +1,14 @@
+`StemContainer` now stores `LineCollection`
+-------------------------------------------
+
+`StemContainer` objects can now store a `LineCollection` object instead of a
+list of `Line2D` objects for stem lines plotted using `ax.stem`. This gives a
+very large performance boost to displaying and moving `ax.stem` plots.
+
+This will become the only behaviour in Matplotlib 3.3. To use it now, the
```

Comment:
Version `3.3` is open to discussion, but I thought I'd put something in as a first go.

Reply:
Do not say that it will become the *only* behavior in v3.3. I suggest that in v3.3, the default value for the `use_line_collection` keyword argument should change from `False` to `True`. Then in v3.6 we can make it a no-op (possibly emitting a warning that it is a no-op, or some other predictable behavior).

Reply:
A no-op doesn't make sense. If someone explicitly uses `use_line_collection=False` they do it because they rely on single Lines in their code. When we remove the functionality, we also have to error out if `use_line_collection != True`. At that time, we can deprecate the argument itself and remove it again later on.